migrations/818-deleted-addons-nullify-guid.py

```diff
@@ -0,0 +1,16 @@
+"""Deleting add-ons is in fact soft deleting them, and emptying some of its
+fields with a unique constraint (like the slug).
+We now also empty the GUID field, to allow re-submitting those add-ons (once
+the GUID has been removed from the BlacklistedGuid through the admin.
+
+This migration empties all the previously soft deleted add-ons."""
+
+import amo
+from addons.models import Addon
+
+
+addons = Addon.unfiltered.filter(status=amo.STATUS_DELETED)
+for addon in addons:
+    amo.log(amo.LOG.DELETE_ADDON, addon.pk, addon.guid, addon)
```

Comment:
Two minor nits:

1) Can you add `created=addon.modified`, just so we have some consistency between old entries and new?
2) We should probably add a comment to the soft deletion code, that if we stop using soft deletion, we need to make sure these entries aren't cascade deleted.